src/de/muenchen/allg/itd51/wollmux/event/GlobalEventListener.java

```diff
@@ -198,6 +198,7 @@ private void onViewCreated(Object source)
     // nicht im docManager mitgeführt werden.
     if (isTempMailMergeDocument(compo))
```

Comment:
Das müsste das gleiche Ergebnis erzielen:
```suggestion
    XTextDocument xTextDoc = UNO.XTextDocument(compo);
    if (xTextDoc != null)
    {
      registerDispatcher(compo.getCurrentController().getFrame());
    }

    if (isTempMailMergeDocument(compo))
    {
      return;
    }
```